eng/common/MaestroHelpers.ps1

```diff
@@ -6,4 +6,25 @@ function ExtractOrgFromAzureDevOpsCollectionUri([string]$CollectionUri)
     $temp1 = $Split1[0]
     $Split2 = $temp1.Substring(8)
     return $Split2
+}
+
+function ConvertToMaestroFriendlyUri([string]$buildRepositoryUri)
+{
+    $devOpsAccount = ExtractOrgFromAzureDevOpsCollectionUri $buildRepositoryUri
+    $buildRepositoryUriSplit = $buildRepositoryUri.Split("/")
+    $repository = "https://dev.azure.com/" + $devOpsAccount + "/" + $buildRepositoryUriSplit[3] + "/" + $buildRepositoryUriSplit[4] + "/" + $buildRepositoryUriSplit[5]
+    return $repository
+}
+
+function IsGitHubRepo([string]$buildRepositoryUri)
+{
+    $githubUri = "https://github.com"
+    if ($buildRepositoryUri.length -ge $githubUri.length)
+    {
+        if($buildRepositoryUri.Substring(0, $githubUri.length) -eq $githubUri)
```

Comment:
>$buildRepositoryUri [](http://example.com/codeflow?start=11&length=19)

is it possible the url is: "[https://www.github.com](https://www.github.com)" or "[http://github.com](http://github.com)"?

Reply:
If it is, there is a security risk and we probably want to fail? 
It should always be https